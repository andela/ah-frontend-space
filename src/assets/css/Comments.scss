.card-width {
  width: 87%;
  margin-left: 10%;
  background-color: none;
  color: black;
  margin-bottom: 13px;
}

.profile-pic {
  border-radius: 50%;
}

.username {
  margin-left: 8px;
  color: gray;
}

.timestamp {
  color: grey;
  margin-left: 8px;
  font-size: small;
}
