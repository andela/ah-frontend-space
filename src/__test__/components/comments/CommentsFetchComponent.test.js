import React from 'react';
import { shallow } from 'enzyme';
import Comments from '../../../components/HomePage/CommentsComponent';


describe('<Comments />', () => {
  const props = {
    comment: {
      comment_body: 'sleek',
      author: {
        username: 'nicks',
      },
    },
  };

  it('it should render component without fail', () => {
    const wrapper = shallow(<Comments {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
