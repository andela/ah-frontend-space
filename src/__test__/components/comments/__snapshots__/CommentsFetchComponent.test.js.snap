// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Comments /> it should render component without fail 1`] = `ShallowWrapper {}`;
